azure_active_directory/README.md

```diff
@@ -0,0 +1,67 @@
+# Agent Check: azure_active_directory
+
+## Overview
+
+Azure Active Directory is a cloud hosted Active Directory offering by Microsoft Azure.
+This integration allows you to ingest your [Azure AD activity logs][1] (Audit and SignIn logs) to Datadog.
+
+## Setup
+
+### Installation
+
+This integration uses log forwarding from Azure using Event hub to Datadog and cofiguring Azure AD to forward activity logs to the event hub.
+
+### Configuration
+
+1. Setup the log forwarding pipeline from Azure to Datadog using Event Hub by following [directions here][2]
+
+2. In Azure portal, Select Azure Active Directory > Monitoring > Audit logs.
+   
+3. Select Export Settings.
+
+4. In the Diagnostics settings pane, do either of the following:
+
+   - To change existing settings, select Edit setting.
+   - To add new settings, select Add diagnostics setting. You can have up to three settings.
+
+5. Select the Stream to an event hub check box, and then select Event Hub/Configure.
+
+6. Select the Azure subscription and Event Hubs namespace that you created earlier to route the logs to.
+   
+7. Select OK to exit the event hub configuration.
+
+8. Do either or both of the following:
+
+   - To send audit logs, select the AuditLogs check box.
+   - To send sign-in logs, select the SignInLogs check box.
+    we recommend that you select both
+  
+9. Select Save to save the setting.
+
+Logs should start coming into the platform after about 15 minutes.
```

Comment:
```suggestion
Logs should start coming into the Datadog within 15 minutes.
```

Reply:
Should probably say "Logs should start coming into Datadog within 15 minutes." (No "the")

Reply:
Changed.